packages/http-crawler/src/internals/http-crawler.ts

```diff
@@ -25,7 +25,8 @@ import contentTypeParser from 'content-type';
 import mime from 'mime-types';
 import type { OptionsInit, Method, Request as GotRequest, Response as GotResponse, GotOptionsInit } from 'got-scraping';
 import { gotScraping, TimeoutError } from 'got-scraping';
-import { extname } from 'path';
+import type { JsonValue } from 'type-fest';
```

Comment:
JSONData is..well, it can realistically be anything when it comes to a JSON API (a number, null, an array, an object, a string), so this just makes it more accurate. It *should* still be easy to work with but if not, we should just replace it with `any` or `unknown`

Reply:
Ok, lets see, worst case we can change it.